client/lib/query-manager/activity/index.js

```diff
@@ -29,8 +29,14 @@ export default class ActivityQueryManager extends QueryManager {
 	 * @return {Number}       0 if equal, less than 0 if itemA is first,
 	 *                        greater than 0 if itemB is first.
 	 */
-	static compare( query, { activityTs: tsA }, { activityTs: tsB } ) {
-		return Date.parse( tsB ) - Date.parse( tsA );
+	static compare( query, a, b ) {
+		if ( a.rewindId && b.rewindId ) {
+			return b.rewindId - a.rewindId;
```

Comment:
Should be good:

https://github.com/Automattic/wp-calypso/blob/37520163ca8373fbcb35fb0a0e3d68f720eb4a48/client/state/data-layer/wpcom/sites/activity/from-api.js#L64

This required by the fromApi schema, so I'd expect this to be present:

https://github.com/Automattic/wp-calypso/blob/8f1319660193905d62435d42805c56d84a5543de/client/state/data-layer/wpcom/sites/activity/schema.json#L37

Reply:
>  `rewind_id` is still optional but

I misread `activityId` in the code 😅